modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -120,7 +121,7 @@ ifdef::openshift-sdn[]
 
   - Projects merged by using the `oc adm pod-network join-projects` command cannot use an egress firewall in any of the joined projects.
 
-* If you create a selectorless service and manually define endpoints or `EndpointSlices` that point to external IPs, traffic to the service IP might still be allowed, even if your `EgressNetworkPolicy` is configured to deny all egress traffic. This occurs because OpenShift SDN does not fully enforce egress network policies for these external endpoints. Consequently, this might result in unexpected access to external services. 
+* If you create a selectorless service and manually define endpoints or `EndpointSlices` that point to external IPs, traffic to the service IP might still be allowed, even if your `EgressNetworkPolicy` is configured to deny all egress traffic. This occurs because OpenShift SDN does not fully enforce egress network policies for these external endpoints. Consequently, this might result in unexpected access to external services.
```

Comment:
```suggestion
* If you create a selectorless service and manually define endpoints or `EndpointSlices` that point to external IPs, traffic to the service IP might still be allowed, even if your egress network policy is configured to deny all egress traffic. This occurs because OpenShift SDN does not fully enforce egress network policies for these external endpoints. Consequently, this might result in unexpected access to external services.
```
EgressNetworkPolicy is not an object in kubernetes

Reply:
I _think_ this is in backticks because it is a resource. @asood-rh

Reply:
@JoeAldinger It is not a resource. That is why I am suggesting the change. These are the resources we have with egress in them.
<pre>
egressfirewalls.k8s.ovn.org                                    
egressips.k8s.ovn.org                                           
egressqoses.k8s.ovn.org                                          
egressrouters.network.operator.openshift.io                  
egressservices.k8s.ovn.org                                       

</pre>

Reply:
Then this is an entirely different bug that you've uncovered. I'm going to fix it quick, standby.